regipy/plugins/ntuser/shellbags_ntuser.py

```diff
@@ -3,18 +3,406 @@
 from regipy.hive_types import NTUSER_HIVE_TYPE
 from regipy.plugins.plugin import Plugin
 from regipy.utils import convert_wintime
+from plaso.helpers.windows import shell_folders
 import re
 
 logger = logging.getLogger(__name__)
 
 NTUSER_SHELLBAG = '\\Software\\Microsoft\\Windows\\Shell\\BagMRU'
-USRCLASS_SHELLBAG = '\\Local Settings\\Software\\Microsoft\\Windows\\Shell\\BagMRU'
 CODEPAGE = 'cp1252'
 
+known_guids = {
```

Comment:
Please move this to a const file besides the plugin.

Reply:
There is specific file to move it to? or to create new one? (and in which path?)

Reply:
Done